From 3772281b8a18b37d5ee50e0dcb1acfad51c55691 Mon Sep 17 00:00:00 2001
From: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Date: Thu, 3 Jul 2014 08:22:48 -0400
Subject: [PATCH] iscsiuio: Rebranding iscsiuio

QLogic has acquired the NetXtremeII products and drivers from Broadcom.
This patch re-brands iscsiuio as QLogic driver.

Signed-off-by: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Acked-by: Eddie Wai <eddie.wai@broadcom.com>
---
 iscsiuio/README                           | 12 ++++-----
 iscsiuio/RELEASE.TXT                      |  9 ++++---
 iscsiuio/configure.ac                     |  3 ++-
 iscsiuio/docs/iscsiuio.8                  |  9 ++++---
 iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.c |  1 +
 iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.h |  1 +
 iscsiuio/src/apps/dhcpc/dhcpv6.c          |  1 +
 iscsiuio/src/apps/dhcpc/dhcpv6.h          |  3 ++-
 iscsiuio/src/uip/icmpv6.h                 |  1 +
 iscsiuio/src/uip/ipv6.c                   |  1 +
 iscsiuio/src/uip/ipv6.h                   |  1 +
 iscsiuio/src/uip/ipv6_ndpc.c              |  1 +
 iscsiuio/src/uip/ipv6_ndpc.h              |  1 +
 iscsiuio/src/uip/ipv6_pkt.h               |  1 +
 iscsiuio/src/uip/uip_eth.c                |  1 +
 iscsiuio/src/unix/iscsid_ipc.c            |  1 +
 iscsiuio/src/unix/iscsid_ipc.h            |  1 +
 iscsiuio/src/unix/libs/bnx2.c             | 17 ++++++------
 iscsiuio/src/unix/libs/bnx2.h             |  1 +
 iscsiuio/src/unix/libs/bnx2x.c            | 45 ++++++++++++++++---------------
 iscsiuio/src/unix/libs/cnic.c             |  1 +
 iscsiuio/src/unix/libs/cnic.h             |  1 +
 iscsiuio/src/unix/logger.c                |  1 +
 iscsiuio/src/unix/logger.h                |  1 +
 iscsiuio/src/unix/nic.c                   |  1 +
 iscsiuio/src/unix/nic.h                   |  1 +
 iscsiuio/src/unix/nic_id.c                |  1 +
 iscsiuio/src/unix/nic_id.h                |  1 +
 iscsiuio/src/unix/nic_nl.c                |  1 +
 iscsiuio/src/unix/nic_nl.h                |  1 +
 iscsiuio/src/unix/nic_utils.c             |  1 +
 iscsiuio/src/unix/nic_utils.h             |  1 +
 iscsiuio/src/unix/nic_vlan.c              |  1 +
 iscsiuio/src/unix/nic_vlan.h              |  1 +
 iscsiuio/src/unix/options.h               |  1 +
 iscsiuio/src/unix/packet.c                |  1 +
 iscsiuio/src/unix/packet.h                |  1 +
 37 files changed, 83 insertions(+), 45 deletions(-)

diff --git a/iscsiuio/README b/iscsiuio/README
index a283116..9ae1411 100644
--- a/iscsiuio/README
+++ b/iscsiuio/README
@@ -3,11 +3,11 @@ Version 0.7.8.2
 Dec 10, 2013
 ------------------------------------------------------
 
-This tool is to be used in conjunction with the Broadcom NetXtreme II Linux
-driver (Kernel module name: 'bnx2' and 'bnx2x'), Broadcom CNIC driver,
-and the Broadcom iSCSI driver (Kernel module name: 'bnx2i').
+This tool is to be used in conjunction with the QLogic NetXtreme II Linux
+driver (Kernel module name: 'bnx2' and 'bnx2x'), QLogic CNIC driver,
+and the QLogic iSCSI driver (Kernel module name: 'bnx2i').
 This user space tool is used in conjunction with the following
-Broadcom Network Controllers:
+QLogic Network Controllers:
   bnx2:  BCM5706, BCM5708, BCM5709 devices
   bnx2x: BCM57710, BCM57711, BCM57711E, BCM57712, BCM57712E,
          BCM57800, BCM57810, BCM57840 devices
@@ -25,8 +25,8 @@ Dependencies:
 =======================================
 
 Linux Kernel Dependencies:
-1.  Broadcom CNIC driver (cnic)
-1.  Broadcom iSCSI offload driver (bnx2i)
+1.  QLogic CNIC driver (cnic)
+1.  QLogic iSCSI offload driver (bnx2i)
 2.  Userspace I/O driver (uio)
 
 Directory Structure of this Package:
diff --git a/iscsiuio/RELEASE.TXT b/iscsiuio/RELEASE.TXT
index 22628fd..19ef717 100644
--- a/iscsiuio/RELEASE.TXT
+++ b/iscsiuio/RELEASE.TXT
@@ -1,13 +1,14 @@
                               Release Notes
-                        Broadcom uIP Linux Driver
+                        QLogic uIP Linux Driver
                             Version 0.7.8.2
                                12/10/2013
 
-                          Broadcom Corporation
-                         5300 California Avenue,
-                            Irvine, CA 92617
+                          QLogic Corporation
+                        26650 Aliso Viejo Pkwy,
+                         Aliso Viejo, CA 92656
 
                Copyright (c) 2004 - 2013 Broadcom Corporation
+                   Copyright (c) 2014, QLogic Corporation
                            All rights reserved
 
 uIP v0.7.8.2 (Dec 10, 2013)
diff --git a/iscsiuio/configure.ac b/iscsiuio/configure.ac
index c7ed6f1..3a47a03 100644
--- a/iscsiuio/configure.ac
+++ b/iscsiuio/configure.ac
@@ -1,6 +1,7 @@
 dnl iscsiuio uIP user space stack configure.ac file
 dnl
 dnl Copyright (c) 2004-2013 Broadcom Corporation
+dnl Copyright (c) 2014, QLogic Corporation
 dnl
 dnl This program is free software; you can redistribute it and/or modify
 dnl it under the terms of the GNU General Public License as published by
@@ -13,7 +14,7 @@ dnl
 PACKAGE=iscsiuio
 VERSION=0.7.8.2
 
-AC_INIT([iscsiuio], [0.7.8.2], [eddie.wai@broadcom.com])
+AC_INIT([iscsiuio], [0.7.8.2], [QLogic-Storage-Upstream@qlogic.com])
 
 AM_INIT_AUTOMAKE([foreign])
 AC_CONFIG_HEADER(config.h)
diff --git a/iscsiuio/docs/iscsiuio.8 b/iscsiuio/docs/iscsiuio.8
index 3b50409..b329979 100644
--- a/iscsiuio/docs/iscsiuio.8
+++ b/iscsiuio/docs/iscsiuio.8
@@ -1,11 +1,12 @@
 .\" Copyright (c) 2010-2013 Broadcom Corporation
+.\" Copyright (c) 2014, QLogic Corporation
 .\" This is free documentation; you can redistribute it and/or
 .\" modify it under the terms of the GNU General Public License as
 .\" published by the Free Software Foundation.
 .\"
 .\" bnx2.4,v 0.7.8.1b
 .\"
-.TH iscsiuio 8 "12/10/2013" "Broadcom Corporation"
+.TH iscsiuio 8 "12/10/2013" "QLogic Corporation"
 .\"
 .\" NAME part
 .\"
@@ -22,9 +23,9 @@ iscsiuio \- iSCSI UserSpace I/O driver
 .\" DESCRIPTION part
 .\"
 .SH DESCRIPTION
-iscsiuio is the UserSpace I/O driver for the Broadcom NetXtreme II
+iscsiuio is the UserSpace I/O driver for the QLogic NetXtreme II
 BCM5706/5708/5709 series PCI/PCI-X Gigabit Ethernet Network Interface Card
-(NIC) and for the Broadcom NetXtreme II BCM57710/57711/57712/57800/57810/57840
+(NIC) and for the QLogic NetXtreme II BCM57710/57711/57712/57800/57810/57840
 series PCI-E 10 Gigabit Ethernet Network Interface Card.
 The driver has been tested on 2.6.28 kernels and above.
 .PP
@@ -84,3 +85,5 @@ Display this help and exit.
 Benjamin Li \- benli@broadcom.com
 .P
 Eddie Wai \- eddie.wai@broadcom.com
+.SH Maintained by
+QLogic-Storage-Upstream@qlogic.com
diff --git a/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.c b/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.c
index 7979774..4223ca4 100644
--- a/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.c
+++ b/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li <benli@broadcom.com>
  *              Based on code example from Adam Dunkels
diff --git a/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.h b/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.h
index f008471..10bfc95 100644
--- a/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.h
+++ b/iscsiuio/src/apps/brcm-iscsi/brcm_iscsi.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li <benli@broadcom.com>
  *              Based on code example from Adam Dunkels
diff --git a/iscsiuio/src/apps/dhcpc/dhcpv6.c b/iscsiuio/src/apps/dhcpc/dhcpv6.c
index a92209a..bfaf269 100644
--- a/iscsiuio/src/apps/dhcpc/dhcpv6.c
+++ b/iscsiuio/src/apps/dhcpc/dhcpv6.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Eddie Wai <eddie.wai@broadcom.com>
  *              Based on code from Kevin Tran's iSCSI boot code
diff --git a/iscsiuio/src/apps/dhcpc/dhcpv6.h b/iscsiuio/src/apps/dhcpc/dhcpv6.h
index f82cfdb..d4ec4b9 100644
--- a/iscsiuio/src/apps/dhcpc/dhcpv6.h
+++ b/iscsiuio/src/apps/dhcpc/dhcpv6.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Eddie Wai <eddie.wai@broadcom.com>
  *              Based on code from Kevin Tran's iSCSI boot code
@@ -227,7 +228,7 @@ struct dhcpv6_option {
 
 #define IANA_ENTERPRISE_NUM_BROADCOM   0x113d
 
-/* Broadcom Extended DHCP options used in iSCSI boot */
+/* QLogic Extended DHCP options used in iSCSI boot */
 #define DHCPV6_TAG_FIRST_ISCSI_TARGET_NAME              201
 #define DHCPV6_TAG_SECOND_ISCSI_TARGET_NAME             202
 #define DHCPV6_TAG_ISCSI_INITIATOR_NAME                 203
diff --git a/iscsiuio/src/uip/icmpv6.h b/iscsiuio/src/uip/icmpv6.h
index 9b00d82..cbf9aa8 100644
--- a/iscsiuio/src/uip/icmpv6.h
+++ b/iscsiuio/src/uip/icmpv6.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by: Eddie Wai  (eddie.wai@broadcom.com)
  *             Based on Kevin Tran's iSCSI boot code
diff --git a/iscsiuio/src/uip/ipv6.c b/iscsiuio/src/uip/ipv6.c
index edaa2e9..cf92bdd 100644
--- a/iscsiuio/src/uip/ipv6.c
+++ b/iscsiuio/src/uip/ipv6.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Eddie Wai  (eddie.wai@broadcom.com)
  *              Based on Kevin Tran's iSCSI boot code
diff --git a/iscsiuio/src/uip/ipv6.h b/iscsiuio/src/uip/ipv6.h
index 91bfaa6..231f454 100644
--- a/iscsiuio/src/uip/ipv6.h
+++ b/iscsiuio/src/uip/ipv6.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Eddie Wai  (eddie.wai@broadcom.com)
  *              Based on Kevin Tran's iSCSI boot code
diff --git a/iscsiuio/src/uip/ipv6_ndpc.c b/iscsiuio/src/uip/ipv6_ndpc.c
index 6f965f4..a396cb7 100644
--- a/iscsiuio/src/uip/ipv6_ndpc.c
+++ b/iscsiuio/src/uip/ipv6_ndpc.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Eddie Wai  (eddie.wai@broadcom.com)
  *              Based on the Swedish Institute of Computer Science's
diff --git a/iscsiuio/src/uip/ipv6_ndpc.h b/iscsiuio/src/uip/ipv6_ndpc.h
index 1eaced9..709a050 100644
--- a/iscsiuio/src/uip/ipv6_ndpc.h
+++ b/iscsiuio/src/uip/ipv6_ndpc.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Eddie Wai  (eddie.wai@broadcom.com)
  *
diff --git a/iscsiuio/src/uip/ipv6_pkt.h b/iscsiuio/src/uip/ipv6_pkt.h
index e0da018..b42f1aa 100644
--- a/iscsiuio/src/uip/ipv6_pkt.h
+++ b/iscsiuio/src/uip/ipv6_pkt.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Eddie Wai  (eddie.wai@broadcom.com)
  *              Based on Kevin Tran's iSCSI boot code
diff --git a/iscsiuio/src/uip/uip_eth.c b/iscsiuio/src/uip/uip_eth.c
index 449616e..9e1ea81 100644
--- a/iscsiuio/src/uip/uip_eth.c
+++ b/iscsiuio/src/uip/uip_eth.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/iscsid_ipc.c b/iscsiuio/src/unix/iscsid_ipc.c
index 033308d..ccd1e67 100644
--- a/iscsiuio/src/unix/iscsid_ipc.c
+++ b/iscsiuio/src/unix/iscsid_ipc.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/iscsid_ipc.h b/iscsiuio/src/unix/iscsid_ipc.h
index 30843fb..60bcd71 100644
--- a/iscsiuio/src/unix/iscsid_ipc.h
+++ b/iscsiuio/src/unix/iscsid_ipc.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/libs/bnx2.c b/iscsiuio/src/unix/libs/bnx2.c
index 91c44e5..937336e 100644
--- a/iscsiuio/src/unix/libs/bnx2.c
+++ b/iscsiuio/src/unix/libs/bnx2.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
@@ -75,20 +76,20 @@ static const char cnic_uio_sysfs_name[] = "bnx2_cnic";
 /*******************************************************************************
  * String constants used to display human readable adapter name
  ******************************************************************************/
-static const char brcm_5706C[] = "Broadcom NetXtreme II BCM5706 1000Base-T";
+static const char brcm_5706C[] = "QLogic NetXtreme II BCM5706 1000Base-T";
 static const char hp_NC370T[] =
 	"HP NC370T Multifunction Gigabit Server Adapter";
 static const char hp_NC370I[] =
 	"HP NC370i Multifunction Gigabit Server Adapter";
-static const char brcm_5706S[] = "Broadcom NetXtreme II BCM5706 1000Base-SX";
+static const char brcm_5706S[] = "QLogic NetXtreme II BCM5706 1000Base-SX";
 static const char hp_NC370F[] =
 	"HP NC370F Multifunction Gigabit Server Adapter";
-static const char brcm_5708C[] = "Broadcom NetXtreme II BCM5708 1000Base-T";
-static const char brcm_5708S[] = "Broadcom NetXtreme II BCM5708 1000Base-SX";
-static const char brcm_5709C[] = "Broadcom NetXtreme II BCM5709 1000Base-T";
-static const char brcm_5709S[] = "Broadcom NetXtreme II BCM5709 1000Base-SX";
-static const char brcm_5716C[] = "Broadcom NetXtreme II BCM5716 1000Base-T";
-static const char brcm_5716S[] = "Broadcom NetXtreme II BCM5716 1000Base-SX";
+static const char brcm_5708C[] = "QLogic NetXtreme II BCM5708 1000Base-T";
+static const char brcm_5708S[] = "QLogic NetXtreme II BCM5708 1000Base-SX";
+static const char brcm_5709C[] = "QLogic NetXtreme II BCM5709 1000Base-T";
+static const char brcm_5709S[] = "QLogic NetXtreme II BCM5709 1000Base-SX";
+static const char brcm_5716C[] = "QLogic NetXtreme II BCM5716 1000Base-T";
+static const char brcm_5716S[] = "QLogic NetXtreme II BCM5716 1000Base-SX";
 
 /*******************************************************************************
  * PCI ID constants
diff --git a/iscsiuio/src/unix/libs/bnx2.h b/iscsiuio/src/unix/libs/bnx2.h
index c025542..3ec9437 100644
--- a/iscsiuio/src/unix/libs/bnx2.h
+++ b/iscsiuio/src/unix/libs/bnx2.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/libs/bnx2x.c b/iscsiuio/src/unix/libs/bnx2x.c
index 748b59d..1268975 100644
--- a/iscsiuio/src/unix/libs/bnx2x.c
+++ b/iscsiuio/src/unix/libs/bnx2x.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
@@ -80,39 +81,39 @@ static const char bnx2x_uio_sysfs_name[] = "bnx2x_cnic";
 /*******************************************************************************
  * String constants used to display human readable adapter name
  ******************************************************************************/
-static const char brcm_57710[] = "Broadcom NetXtreme II BCM57710 10-Gigabit";
-static const char brcm_57711[] = "Broadcom NetXtreme II BCM57711 10-Gigabit";
-static const char brcm_57711e[] = "Broadcom NetXtreme II BCM57711E 10-Gigabit";
-static const char brcm_57712[] = "Broadcom NetXtreme II BCM57712 10-Gigabit";
-static const char brcm_57712_MF[] = "Broadcom NetXtreme II BCM57712 MF "
+static const char brcm_57710[] = "QLogic NetXtreme II BCM57710 10-Gigabit";
+static const char brcm_57711[] = "QLogic NetXtreme II BCM57711 10-Gigabit";
+static const char brcm_57711e[] = "QLogic NetXtreme II BCM57711E 10-Gigabit";
+static const char brcm_57712[] = "QLogic NetXtreme II BCM57712 10-Gigabit";
+static const char brcm_57712_MF[] = "QLogic NetXtreme II BCM57712 MF "
 				    "10-Gigabit";
-static const char brcm_57712_VF[] = "Broadcom NetXtreme II BCM57712 VF "
+static const char brcm_57712_VF[] = "QLogic NetXtreme II BCM57712 VF "
 				    "10-Gigabit";
-static const char brcm_57713[] = "Broadcom NetXtreme II BCM57713 10-Gigabit";
-static const char brcm_57713e[] = "Broadcom NetXtreme II BCM57713E 10-Gigabit";
-static const char brcm_57800[] = "Broadcom NetXtreme II BCM57800 10-Gigabit";
-static const char brcm_57800_MF[] = "Broadcom NetXtreme II BCM57800 MF "
+static const char brcm_57713[] = "QLogic NetXtreme II BCM57713 10-Gigabit";
+static const char brcm_57713e[] = "QLogic NetXtreme II BCM57713E 10-Gigabit";
+static const char brcm_57800[] = "QLogic NetXtreme II BCM57800 10-Gigabit";
+static const char brcm_57800_MF[] = "QLogic NetXtreme II BCM57800 MF "
 				    "10-Gigabit";
-static const char brcm_57800_VF[] = "Broadcom NetXtreme II BCM57800 VF "
+static const char brcm_57800_VF[] = "QLogic NetXtreme II BCM57800 VF "
 				    "10-Gigabit";
-static const char brcm_57810[] = "Broadcom NetXtreme II BCM57810 10-Gigabit";
-static const char brcm_57810_MF[] = "Broadcom NetXtreme II BCM57810 MF "
+static const char brcm_57810[] = "QLogic NetXtreme II BCM57810 10-Gigabit";
+static const char brcm_57810_MF[] = "QLogic NetXtreme II BCM57810 MF "
 				    "10-Gigabit";
-static const char brcm_57810_VF[] = "Broadcom NetXtreme II BCM57810 VF "
+static const char brcm_57810_VF[] = "QLogic NetXtreme II BCM57810 VF "
 				    "10-Gigabit";
-static const char brcm_57811[] = "Broadcom NetXtreme II BCM57811 10-Gigabit";
-static const char brcm_57811_MF[] = "Broadcom NetXtreme II BCM57811 MF "
+static const char brcm_57811[] = "QLogic NetXtreme II BCM57811 10-Gigabit";
+static const char brcm_57811_MF[] = "QLogic NetXtreme II BCM57811 MF "
 				    "10-Gigabit";
-static const char brcm_57811_VF[] = "Broadcom NetXtreme II BCM57811 VF "
+static const char brcm_57811_VF[] = "QLogic NetXtreme II BCM57811 VF "
 				    "10-Gigabit";
-static const char brcm_57840[] = "Broadcom NetXtreme II BCM57840 10-Gigabit";
-static const char brcm_57840_MF[] = "Broadcom NetXtreme II BCM57840 MF "
+static const char brcm_57840[] = "QLogic NetXtreme II BCM57840 10-Gigabit";
+static const char brcm_57840_MF[] = "QLogic NetXtreme II BCM57840 MF "
 				    "10-Gigabit";
-static const char brcm_57840_VF[] = "Broadcom NetXtreme II BCM57840 VF "
+static const char brcm_57840_VF[] = "QLogic NetXtreme II BCM57840 VF "
 				    "10-Gigabit";
-static const char brcm_57840_4_10[] = "Broadcom NetXtreme II BCM57840 4x"
+static const char brcm_57840_4_10[] = "QLogic NetXtreme II BCM57840 4x"
 				      "10-Gigabit";
-static const char brcm_57840_2_20[] = "Broadcom NetXtreme II BCM57840 2x"
+static const char brcm_57840_2_20[] = "QLogic NetXtreme II BCM57840 2x"
 				      "20-Gigabit";
 
 /*******************************************************************************
diff --git a/iscsiuio/src/unix/libs/cnic.c b/iscsiuio/src/unix/libs/cnic.c
index 6ba8bce..7f473c4 100644
--- a/iscsiuio/src/unix/libs/cnic.c
+++ b/iscsiuio/src/unix/libs/cnic.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by: Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/libs/cnic.h b/iscsiuio/src/unix/libs/cnic.h
index 3e53307..6244a94 100644
--- a/iscsiuio/src/unix/libs/cnic.h
+++ b/iscsiuio/src/unix/libs/cnic.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/logger.c b/iscsiuio/src/unix/logger.c
index 1a65793..d41f9e8 100644
--- a/iscsiuio/src/unix/logger.c
+++ b/iscsiuio/src/unix/logger.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by: Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/logger.h b/iscsiuio/src/unix/logger.h
index edf85ae..06e2084 100644
--- a/iscsiuio/src/unix/logger.h
+++ b/iscsiuio/src/unix/logger.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic.c b/iscsiuio/src/unix/nic.c
index 7c3f4d5..9dea1a6 100644
--- a/iscsiuio/src/unix/nic.c
+++ b/iscsiuio/src/unix/nic.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic.h b/iscsiuio/src/unix/nic.h
index 7d1ae28..8484032 100644
--- a/iscsiuio/src/unix/nic.h
+++ b/iscsiuio/src/unix/nic.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_id.c b/iscsiuio/src/unix/nic_id.c
index 8f21ba9..6b2467c 100644
--- a/iscsiuio/src/unix/nic_id.c
+++ b/iscsiuio/src/unix/nic_id.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_id.h b/iscsiuio/src/unix/nic_id.h
index 8d1e1db..340580f 100644
--- a/iscsiuio/src/unix/nic_id.h
+++ b/iscsiuio/src/unix/nic_id.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_nl.c b/iscsiuio/src/unix/nic_nl.c
index 3ab4eb9..391003f 100644
--- a/iscsiuio/src/unix/nic_nl.c
+++ b/iscsiuio/src/unix/nic_nl.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_nl.h b/iscsiuio/src/unix/nic_nl.h
index 90f76be..c68d81c 100644
--- a/iscsiuio/src/unix/nic_nl.h
+++ b/iscsiuio/src/unix/nic_nl.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_utils.c b/iscsiuio/src/unix/nic_utils.c
index 2acd3c2..06e8f30 100644
--- a/iscsiuio/src/unix/nic_utils.c
+++ b/iscsiuio/src/unix/nic_utils.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_utils.h b/iscsiuio/src/unix/nic_utils.h
index 2644e43..962d130 100644
--- a/iscsiuio/src/unix/nic_utils.h
+++ b/iscsiuio/src/unix/nic_utils.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_vlan.c b/iscsiuio/src/unix/nic_vlan.c
index d371ec5..eb33381 100644
--- a/iscsiuio/src/unix/nic_vlan.c
+++ b/iscsiuio/src/unix/nic_vlan.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/nic_vlan.h b/iscsiuio/src/unix/nic_vlan.h
index f7293d4..610f721 100644
--- a/iscsiuio/src/unix/nic_vlan.h
+++ b/iscsiuio/src/unix/nic_vlan.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/options.h b/iscsiuio/src/unix/options.h
index 9691c4e..df03255 100644
--- a/iscsiuio/src/unix/options.h
+++ b/iscsiuio/src/unix/options.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/packet.c b/iscsiuio/src/unix/packet.c
index 1657d73..efb3eec 100644
--- a/iscsiuio/src/unix/packet.c
+++ b/iscsiuio/src/unix/packet.c
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
diff --git a/iscsiuio/src/unix/packet.h b/iscsiuio/src/unix/packet.h
index f62b685..b63d688 100644
--- a/iscsiuio/src/unix/packet.h
+++ b/iscsiuio/src/unix/packet.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2009-2011, Broadcom Corporation
+ * Copyright (c) 2014, QLogic Corporation
  *
  * Written by:  Benjamin Li  (benli@broadcom.com)
  *
-- 
2.5.0

